src/Common/tests/System/Runtime/Serialization/Formatters/BinaryFormatterHelpers.cs

```diff
@@ -36,7 +36,10 @@ public static void AssertRoundtrips<T>(T expected, params Func<T, object>[] addi
 				T actual = Clone(expected);
 
 				// Verify core state
-				Assert.Equal(expected.StackTrace, actual.StackTrace);
+				if (!PlatformDetection.IsFullFramework) // On full framework, line number is method body start
```

Comment:
The build is failing because this class is in a file that's not included in all projects that use this file.